src/main/java/seedu/address/logic/commands/DeleteEventCommand.java

```diff
@@ -0,0 +1,56 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.event.Event;
+
+/**
+ * Deletes an Event from the AddressBook based on the index supplied
+ */
+public class DeleteEventCommand extends Command {
+
+    public static final String COMMAND_WORD = "deleteEvent";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes the Event based on the index number in the Event list displayed\n"
+            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Example: "
+            + COMMAND_WORD
+            + " 3";
+
+    public static final String MESSAGE_DELETED_EVENT_SUCCESS = "Deleted Event: %1$s";
+
+    private final Index targetEventIndex;
+
+    public DeleteEventCommand(Index targetEventIndex) {
```

Comment:
Add javadoc in constructor to be consistent with AddEvent command

Reply:
ok addded